youtube_dl/__init__.py

```diff
@@ -169,7 +169,7 @@ def _real_main(argv=None):
         if not opts.audioquality.isdigit():
             parser.error('invalid audio quality specified')
     if opts.recodevideo is not None:
-        if opts.recodevideo not in ['mp4', 'flv', 'webm', 'ogg', 'mkv']:
+        if opts.recodevideo not in ['mp4', 'flv', 'webm', 'ogg', 'mkv', 'xvid']:
```

Comment:
@svnpenn thanks for your proposal, but... i'm on a country well known as "third world"; this model of Smart TV is used by schools; and if i have money to throw away a TV and buy a new one, still that is a really bad action against the planet. So... i understand your good intention, but better software can save money, classrooms and the blue marble ball. :-)